expense_app/admin.py

```diff
@@ -0,0 +1,3 @@
+# from django.contrib import admin
```

Comment:
Why is this import statement commented out?

Reply:
because i didnt get to the point of setting up the admin.py file , and my commit failed flake8 because the import wasnt used